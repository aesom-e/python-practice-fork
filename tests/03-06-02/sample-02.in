Cryptography 101: Rule Z?
-1
-2
5