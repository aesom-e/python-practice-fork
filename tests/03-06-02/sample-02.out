Bpdortfpfofd 101: Qsqd X?